tests/unit/test-utils.js

```diff
@@ -0,0 +1,10 @@
+import Rx from 'rxjs';
+
+export function waitFor(fn) {
```

Comment:
Tell me if you prefer class with static methods instead. I found directly exported functions even more handy, but I don't have strong opinion.